script/client

```diff
@@ -158,9 +158,11 @@ sub usage($) {
 }
 
 sub handle_result {
-    my $res = shift;
+    my $res     = shift;
+    my $rescode = $res->code // 0;
 
-    if (($res->code // 0) == 200) {
+    if ($rescode >= 200 && $rescode <= 299) {
+        printf(STDERR "%s - %s\n", $rescode, $res->{error}->{message}) if $rescode > 200;
```

Comment:
And you are sure `$res->{error}` and `$res->{error}->{message}` is populated in all possible errors? I would checket that to prevent warnings.

Reply:
The original code doesn't do it, so I wasn't sure why and if the check is really needed here. Do you know why - or should I just add the check?

Reply:
Should I add check for `$res->body` few lines later as well?

Reply:
Ah, you're referring to the other `printf` line below. I'd prefer to check whether the error message is present in all cases.